### PR TITLE
Last minute updates

### PR DESCRIPTION
**Pull request by vincenthz**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    mkdir -p ${ENV_PREFIX}/src && \
+    mkdir -p ${ENV_PREFIX}/bin && \
+    bash -c "curl https://sh.rustup.rs -sSf | bash -s -- -y" && \
+    export PATH=$HOME/.cargo/bin:$PATH && \
+    rustup install stable && \
+    rustup default stable && \
+    cd ${ENV_PREFIX} && \
+    git clone --recurse-submodules https://github.com/input-output-hk/jormungandr src && \
+    cd src && \
+    git checkout -b last-minutes-updates && \
```
Comment (mzabaluev) [inline docker/Dockerfile]: Why switch to the new branch?

Comment (onicrom) [reply docker/Dockerfile]: Why not make the branch a --build-arg when you build the docker container:

# default git branch 
ARG BRANCH=master
ENV ENV_BRANCH=${BRANCH}

git checkout -b ${ENV_BRANCH}

During the docker build  you can specify --build-arg BRANCH=last-minutes-updates

**Comment by onicrom** [comment]
might be best to use ubuntu:bionic instead of the non LTS release of ubuntu.

**Comment by mzabaluev** [comment]
> might be best to use ubuntu:bionic instead of the non LTS release of ubuntu.

We are Rust people, we need latest and greatest :)
Seriously though, as the toolchain is pulled by rustup, I don't know if we need anything really new.

File: docker/Dockerfile (hunk 1 above)
Comment (vincenthz) [reply docker/Dockerfile]: damn, I forgot to remove it

Comment (vincenthz) [reply docker/Dockerfile]: I briefly look at your script btw @onicrom, I just pulled really quickly couple of improvement inspired by your docker, to the one I had locally but clearly this is being thrown together. down the line there's multiple improvements to make to the script, and make multiple variations

**Comment by vincenthz** [comment]
no reason for choosing the ubuntu release, we don't need much from the system anyway. Also I don't expect this docker file to be really production ready; it's more for getting some docker stuff in the repo than a final product at this stage.

**Pull request merged**